packages/useful-types/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 ### Added
 
 - Added `PartialSome` and `RequireSome` types to set specified fields of a property to optional or required
+- Use Typescript native types and pave the ground to deprecate legacy types
```

Comment:
This can be removed, as we're no longer touching useful-types

Reply:
Done